### PR TITLE
Feature/194 tribal name drinking water

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* [HMW-194](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-194&quickFilter=2479)

## Main Changes:
* Added the `tribal_name` and `tribal_code` properties from the drinking water data to the accordions on the Drinking Water tab, as the fields *Tribal Name* and *Tribal ID*, respectively. The fields are visible only if there are values for the properties.
* Added a **Tribal Only** filter switch on both the providers and the withdrawers sub-tabs. It is visible in each only if there exists tribal data.

## Steps To Test:
1. Go to the **Drinking Water** tab of the **Community** page, and find a watershed with tribal data, for example  [http://localhost:3000/community/130202010611/drinking-water](http://localhost:3000/community/130202010611/drinking-water)
2. In the case of the example, there is only one tribal withdrawer, so click "Who withdraws water for drinking here?"
3. Toggle the **Tribal Only** switch, and open the accordion of the displayed system
4. The accordion item should have the fields *Tribal Name* and *Tribal ID*

### CODE REVIEW

File: app/client/src/components/pages/Community.Tabs.DrinkingWater.js [hunk 1]
```diff
@@ -640,26 +681,65 @@ function DrinkingWater() {
...
-                          sortOptions={providerSorts}
-                        >
-                          {sortWaterSystems(
-                            providers,
-                            providersSortBy,
-                            false,
-                          ).map((item) => createAccordionItem(services, item))}
-                        </AccordionList>
+                        <>
+                          {tribalProviderCount > 0 && (
```
Comment (cschwinderg) [inline app/client/src/components/pages/Community.Tabs.DrinkingWater.js:685]: I think the Tribal Status switches should always be visible and just be disabled if the `tribalProviderCount` equals 0. This is what we do for the Ground Water and Surface Water switches.

File: app/client/src/components/pages/Community.Tabs.DrinkingWater.js [hunk 2]
```diff
@@ -826,6 +909,34 @@ function DrinkingWater() {
                                 </tr>
                               )}
                             </tbody>
+                            {tribalWithdrawerCount > 0 && (
```
Comment (cschwinderg) [inline app/client/src/components/pages/Community.Tabs.DrinkingWater.js:912]: Same as above. The Tribal Status switch should always be visible and just disabled if the `tribalWithdrawerCount` equals 0.

Comment (maxdiebold-erg) [reply app/client/src/components/pages/Community.Tabs.DrinkingWater.js:912]: Sounds good, yeah I wasn't sure about that.

Comment (cschwinderg) [reply app/client/src/components/pages/Community.Tabs.DrinkingWater.js]: Thanks for making that change.

**Pull request merged**